### ISSUE TITLE
how can I change top nav icons like twitter github etc.

### ISSUE THREAD

**Issue by abalta** [sandrokeil/ghost-theme-casperion#8]
I have replaced my own icomoon folder to yours, and edit shortscodes. Nothing happens.

**Comment by sandrokeil** [comment]
You have to do the following:

1. Create your own font e.g. with iconmoon
1. update [_icons.scss](https://github.com/sandrokeil/ghost-theme-casperion/blob/master/src/assets/css/partials/_icons.scss) with you icons and font
1. Update the [navigation.hbs](https://github.com/sandrokeil/ghost-theme-casperion/blob/master/partials/navigation.hbs) with your icons
1. Copy the font to the `assets/fonts` folder
1. run `gulp` so that the new CSS/JS files will be compiled to the assets folder

**Comment by abalta** [comment]
How can I run "gulp"? I have never work gulp before.

**Comment by sandrokeil** [comment]
@abalta Yes you must install [npm](https://www.npmjs.com/), [gulp](https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md) and the dependencies.
